CMSgov/onemac
# OY2 22487: Waiver rai handling is missing in the package builder, RAI's missing after migration.

Story: https://qmacbis.atlassian.net/browse/OY2-22487
Endpoint: See github-actions bot comment

### Details
Migration isn't migrating the Waiver RAI's properly - need to include parentType attribute or the builder does not know what to build.

### Changes
- Used the Submission View "type" parsers to figure out the type of the parent package

### Test Plan
1. Log in as a submitting user
2. Navigate to Submission Dashboard
3. Submit a Waiver RAI Response
4. Use the AWS console to run a convertChangeRequests lambda (until a merge-back adds the scheduling from develop)
5. Verify the Waiver RAI is properly represented on the Package Dashboard/Waiver tab
